langgraph_prototype/tests/integration/test_health_check.py

```
@@ -0,0 +1,248 @@
+        messages=[{"role": "user", "content": "こんにちは"}],
+        user_id="health_perf_user",
+        thread_id="health_perf_thread",
+        next_agent="",
+        pending_confirmation={},
+        final_response="",
+        intermediate_steps=[]
+    )
+    
+    try:
+        start_time = time.time()
+        perf_result = await langgraph_app.ainvoke(perf_state)
```

Ruff (0.12.2)

Local variable `perf_result` is assigned to but never used

Remove assignment to unused variable `perf_result`

(F841)